docker/accumulo/Dockerfile

```diff
@@ -47,13 +52,13 @@ ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64/
 COPY ./files/ .
 # Otherwise, download official distributions
 RUN if [ ! -f "./accumulo-${ACCUMULO_VERSION}-bin.tar.gz" ]; then \
-		curl -s -fLO ${ACCUMULO_DOWNLOAD_URL}; \
+		(wget -nv -O ./accumulo-${ACCUMULO_VERSION}-bin.tar.gz ${ACCUMULO_DOWNLOAD_URL} || wget -nv -O ./accumulo-${ACCUMULO_VERSION}-bin.tar.gz ${ACCUMULO_BACKUP_DOWNLOAD_URL}); \
```

Comment:
Just wondering why the change from `curl` to `wget`?

Reply:
It was just so that we would have a consistent command across dockerfiles